src/FileReader/FileLocator.php

```diff
@@ -5,18 +5,12 @@
 final class FileLocator
 {
     /**
-     * @param array $patterns
+     * @param string $pattern
      *
      * @return string[]
      */
-    public function locate(array $patterns)
+    public function locate($pattern)
     {
-        $files = [];
-
-        foreach ($patterns as $pattern) {
-            $files = array_merge($files, glob($pattern, GLOB_BRACE));
-        }
-
-        return $files;
+        return glob($pattern, GLOB_BRACE);
```

Comment:
@tomphp 

As suggested in https://github.com/tomphp/config-service-provider/pull/37/files?short_path=04c6e90#r77788264, should we assert that `$pattern` is a `string` here before passing it to `glob()`?